Ports/.hosted_defs.sh

```diff
@@ -10,8 +10,8 @@ export PKG_CONFIG_DIR=""
 export PKG_CONFIG_SYSROOT_DIR="${SERENITY_BUILD_DIR}/Root"
 export PKG_CONFIG_LIBDIR="${PKG_CONFIG_SYSROOT_DIR}/usr/lib/pkgconfig/:${PKG_CONFIG_SYSROOT_DIR}/usr/local/lib/pkgconfig"
 
-# To be removed.
-export SERENITY_ROOT="$(realpath "${SCRIPT}/../")"
+# To be deprecated soon.
+export SERENITY_SOURCE_DIR="$(realpath "${SCRIPT}/../")"
```

Comment:
is this intentional? we already define that above and don't want to deprecate it :D